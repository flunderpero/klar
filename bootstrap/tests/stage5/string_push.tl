let s = "Hi"
let i = 42

s.push(" ")
s.push("there")
print(s)

s.push(" ").push(i.to_string()).push("!")
print(s)
