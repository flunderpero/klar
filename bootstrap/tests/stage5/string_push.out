Hi there
Hi there 42!
